hybrid/SampleApps/NoteSync/NoteSync/SFContentSoqlSyncDownTarget.m

```diff
@@ -274,24 +290,12 @@ + (SFContentSoqlSyncDownTarget*) newSyncTarget:(NSString*)query {
 }
 
 
-#pragma mark - From/to dictionary
-
-+ (SFContentSoqlSyncDownTarget*) newFromDict:(NSDictionary*)dict {
-    SFContentSoqlSyncDownTarget* syncTarget = nil;
-    if (dict != nil && [dict count] != 0) {
-        syncTarget = [[SFContentSoqlSyncDownTarget alloc] init];
-        syncTarget.queryType = SFSyncDownTargetQueryTypeCustom;
-        syncTarget.query = dict[kSFSoqlSyncTargetQuery];
-    }
-    return syncTarget;
-}
+#pragma mark - To dictionary
 
-- (NSDictionary*) asDict {
-    return @{
-             kSFSyncTargetTypeKey: [SFSyncDownTarget queryTypeToString:self.queryType],
-             kSFSoqlSyncTargetQuery: self.query,
-             kSFSyncTargetiOSImplKey: NSStringFromClass([self class])
-             };
+- (NSMutableDictionary*) asDict {
+    NSMutableDictionary *dict = [super asDict];
```

Comment:
`asDict` will use this model to inherit property serialization from the class's parent.